### PR TITLE
Teams form renaming to enterprise

### PR DESCRIPTION
**Pull request by mitchmac**

### CODE REVIEW

**Comment by DirtyF** [comment]
@mitchmac FWIW the title in the browser tab is still "Teams" not Enterprise.

File: content/pages/teams.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "title": "Teams",
+  "title": "Enterprise",
   "description": "The power of Tina for the entire team.",
```
Comment (DirtyF) [inline content/pages/teams.json:3]:
```suggestion
  "description": "The power of Tina for your teams.",
```

File: content/pages/teams.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
   "headline": "The power of Tina for the entire team",
```
Comment (DirtyF) [inline content/pages/teams.json:4]:
```suggestion
  "headline": "The power of Tina for the entire company",
```

Comment (DirtyF) [reply content/pages/teams.json:4]: More company-friendly?

Comment (mitchmac) [reply content/pages/teams.json:4]: There's some more content changes coming from Dan in a follow-up.

**Pull request merged**